### PR TITLE
fetch email and phone number on authentication

### PR DESCRIPTION
**Pull request by silicoflare**
Along with the usual details being fetched, the API now fetches and returns the email address and phone number of the student. This feature will be useful in the apps we plan to build in the future.

### CODE REVIEW

**Comment by silicoflare** [comment]
@aditeyabaral @Icarus131 please lmk if there are any changes, thanks!

**Comment by aditeyabaral** [comment]
@silicoflare 
Just a comment: Ideally, this API is meant for authentication only, with minimum details returned from it to render placeholders like name, branch, etc. This is just an authentication gateway (to keep things quick), not a way to extract details.

If you are looking to develop apps with the backend APIs where there is a focus on reading details from the website, I would suggest contributing to this: https://github.com/aditeyabaral/pesu-academy-api

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,6 @@ github-markdown-css/
 images/
 *.log
 README.html
-README_tmp*
\ No newline at end of file
+README_tmp*
+
```
Comment (aditeyabaral) [inline .gitignore:9]: You can remove this extra line

File: app/pesu.py [hunk 2]
```diff
@@ -145,6 +145,9 @@ def get_profile_information_from_requests(
                 key = "prn" if key == "pesu_id" else key
                 profile[key] = value
 
+        profile["email"] = soup.find("input", attrs={"id": "updateMail"})["value"]
+        profile["phone"] = soup.find("input", attrs={"id": "updateContact"})["value"]
```
Comment (aditeyabaral) [inline app/pesu.py:149]: Handle missing keys:
```python
profile["email"] = soup.find("input", attrs={"id": "updateMail"}).get("value")
```
Same for mobile.

**Review by aditeyabaral — CHANGES_REQUESTED** [review]

**Comment by aditeyabaral** [comment]
You also need to update the README with the new data you are returning.

**Comment by silicoflare** [comment]
> @silicoflare 
> Just a comment: Ideally, this API is meant for authentication only, with minimum details returned from it to render placeholders like name, branch, etc. This is just an authentication gateway (to keep things quick), not a way to extract details.
> 
> If you are looking to develop apps with the backend APIs where there is a focus on reading details from the website, I would suggest contributing to this: https://github.com/aditeyabaral/pesu-academy-api 


So here's the thing: in pesu-tix, when the user signs in using PESU Auth, I want to fetch all these details and store it in a database, so that when the student registers for the event, all those details will be added to the event database record, which cuts the need of the user to fill any additional forms.

**Comment by silicoflare** [comment]
> You also need to update the README with the new data you are returning.

Yes will do, thanks!

**Comment by aditeyabaral** [comment]
> > @silicoflare
> > Just a comment: Ideally, this API is meant for authentication only, with minimum details returned from it to render placeholders like name, branch, etc. This is just an authentication gateway (to keep things quick), not a way to extract details.
> > If you are looking to develop apps with the backend APIs where there is a focus on reading details from the website, I would suggest contributing to this: https://github.com/aditeyabaral/pesu-academy-api
> 
> So here's the thing: in pesu-tix, when the user signs in using PESU Auth, I want to fetch all these details and store it in a database, so that when the student registers for the event, all those details will be added to the event database record, which cuts the need of the user to fill any additional forms.

Alright, but no more fields will be added (note to future PR raisers). Please address the requested changes and push a commit.

**Comment by silicoflare** [comment]
@aditeyabaral i've made the requested changes, please do check!

**Review by aditeyabaral — APPROVED** [review]

**Pull request merged**